src/CloudFoundry.VisualStudio/VsUtils.cs

```diff
@@ -114,12 +117,13 @@ public static Project GetSelectedProject()
             return GetProjectFromSelection();
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Prevent crash if bad msbuild package")]
```

Comment:
An exception here could mean that the nuget service isn't working (for various reasons).